build/integration.sh

```diff
@@ -39,6 +39,7 @@ ENVOY_PID=$!
 function cleanup() {
   kill ${ENVOY_PID} ${UPSTREAM_PID}
   wait ${ENVOY_PID} ${UPSTREAM_PID} 2> /dev/null || true
+  cat ${ENVOY_LOG}
```

Comment:
Ah can you remove this cat? We don't want to pollute this log here

Reply:
oops, I forgot about this, this is fixed now.

Reply:
Awesome thank you!